src/app/logic/player.js

```diff
@@ -0,0 +1,44 @@
+export class Player {
+  constructor() {
+    this.correctAnswersScore = 0;
+    this.amountOfAnsweredQuestions = 0;
+    this.detailQuestionData = [];
+    this.endGameData = {};
+  }
+
+  registerAnswer(correctAnswer, playerAnswer, isanswerCorrect) {
+    if (this.playerAnswer === false) {
+      throw new Error(
+        'error with playerAnswer, playerAnswer is null or undefined',
```

Comment:
There is also a list of other falsy values: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
thanks! so useful to have in one place all falsy values

Reply:
very helpful document!